modules/nw-rfhe-subscribing-redfish-events.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * networking/using-rfhe.adoc
+
+[id="nw-rfhe-troubleshooting_{context}"]
+= Subscribing to Redfish Events
+
+To enable the Redfish Event Service, set the ServiceEnabled property in the EventService URI (/redfish/v1/EventService) to Enabled. <is this correct? If not, what is correct?>
+
+To subscribe to Redfish Events, create a BMCEventSubscription CRD (https://github.com/mettehal3-io/metal3-docs/pull/167 - link gets 404).
+
+[source,yaml]
+----
+    apiVersion: metal3.io/v1alpha1
+    kind: BMCEventSubscription
+    metadata:
+      name: worker-1-events
+    spec:
+       hostRef: ostest-worker-1 <1>
+       destination: https://events.apps.corp.example.com/webhook
+       context: “SomeUserContext”
+       eventTypes:
+       - Alert
+       protocol: Redfish
+    status:
+      errorMessage: ""
+      errorCount: 0
+      subscriptionID: aa618a32-9335-42bc-a04b-20ddeed13ade
+----
+
+<1> hostRef is the name or uuid of the worker node where redfish events are generated.
```

Comment:
hostRef needs backticks.
I'd assume that uuid needs backticks as well and I think it should be expanded.
redfish should be capitalized